howtos/how-to-make-a-threaded-chat/server.js

```diff
@@ -0,0 +1,228 @@
+import express from "express";
+import cors from "cors";
+import fetch from "node-fetch";
+
+const appId = "<APP_ID>";
+const secretKey = "<SECRET_KEY>";
+
+const basePath = "https://api.talkjs.com";
+
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+app.listen(3000, () => console.log("Server is up"));
+
+const senderId = `threadsExampleSender`;
+const receiverId = `threadsExampleReceiver`;
+
+// Create a thread as a new conversation
+async function createThread(parentMessageId, parentConvId, participants) {
+  const conversationId = "replyto_" + parentMessageId;
```

Comment:
It's a bit confusing that we use both concatenation and interpolation in the same code same for using both backticks and double quotes

Reply:
Switched to all interpolation, that's definitely better